TrafficCapture/dockerSolution/src/main/docker/migrationConsole/Dockerfile

```diff
@@ -42,21 +42,23 @@ RUN chmod ug+x /root/loadServicesFromParameterStore.sh
 WORKDIR /root
 COPY lib /root/lib
 WORKDIR /root/lib/console_link
-RUN pipenv install --deploy --ignore-pipfile
+RUN pipenv install --deploy
```

Comment:
This change is what causes it to throw an error if the lockfile is out of date?

Reply:
Yes. --deploy and --ignore-pipfile are contradictory and the --ignore-pipfile took precedence.

--deploy is what we want. Install from Pipfile.lock, but fail if it doesn't reflect the Pipfile